Makefile

```diff
@@ -11,6 +14,12 @@ serve:
 
 todo:
 	tools/mdbuild.py --todo
+
+.venv: .venv/bin/python
+.venv/bin/python:
+	virtualenv -p python3 .venv
+	$(PIP) install html2text Markdown mdx-anchors-away mdx-callouts mdx-foldouts linkchecker
+
 sysdeps:
```

Comment:
Thanks for doing the virtualenv. Shouldn't this sysdeps target be deleted/updated?